src/js/models/v2/aboutTheData/CoreReportingRow.js

```diff
@@ -8,25 +8,25 @@ import { format } from 'date-fns';
 
 const CoreReportingRow = {
     populateCore(data) {
-        this._budgetAuthority = data.current_total_budget_authority_amount || 0;
+        this._budgetAuthority = data.current_total_budget_authority_amount;
         this._mostRecentPublicationDate = data.recent_publication_date || null;
         /* eslint-disable camelcase */
-        this._gtasObligationTotal = data.tas_account_discrepancies_totals?.gtas_obligation_total || 0;
-        this._discrepancyCount = data.tas_account_discrepancies_totals?.missing_tas_accounts_count || 0;
+        this._gtasObligationTotal = data.tas_account_discrepancies_totals?.gtas_obligation_total;
+        this._discrepancyCount = data.tas_account_discrepancies_totals?.missing_tas_accounts_count;
         /* eslint-enable camelcase */
-        this._obligationDifference = data.obligation_difference || 0;
-        this._unlinkedContracts = data.unlinked_contract_award_count || 0;
-        this._unlinkedAssistance = data.unlinked_assistance_award_count || 0;
+        this._obligationDifference = data.obligation_difference;
+        this._unlinkedContracts = data.unlinked_contract_award_count;
```

Comment:
Since we removed the `|| 0` conditional are we handling these values if the come back null?

Reply:
yes

Reply:
cool, do you know where?

The core reporting row has getters that call format number, and none of the parent row objects are handling them.
```
    get unlinkedContracts() {
        return formatNumber(this._unlinkedContracts);
    },
    get unlinkedAssistance() {
        return formatNumber(this._unlinkedAssistance);
    }
```


And in the agencies container we are looking for zero and deciding on whether or not to render a modal cell. But with the current code it looks like we will just display null if the api comes back with null?

```
            unlinkedContracts !== '0' ? (<CellWithModal
                data={unlinkedContracts}
                openModal={openModal}
                modalType="unlinkedData"
                agencyData={{
                    agencyName,
                    agencyCode: code,
                    fiscalYear: selectedFy,
                    fiscalPeriod: selectedPeriod?.id,
                    type: 'Contract'
                }} />) : (<div className="generic-cell-content">{unlinkedContracts}</div>),
            unlinkedAssistance !== '0' ? (<CellWithModal
                data={unlinkedAssistance}
                openModal={openModal}
                modalType="unlinkedData"
                agencyData={{
                    agencyName,
                    agencyCode: code,
                    fiscalYear: selectedFy,
                    fiscalPeriod: selectedPeriod?.id,
                    type: 'Assistance'
                }} />) : (<div className="generic-cell-content">{unlinkedAssistance}</div>),
```

Reply:
cool, do you know where?

currently the core reporting row only has the getters below, and the parent objects to the core reporting row are not touching these fields.

```
    get unlinkedContracts() {
        return formatNumber(this._unlinkedContracts);
    },
    get unlinkedAssistance() {
        return formatNumber(this._unlinkedAssistance);
    }
```

The agencies container is only looking for zero and handling displaying a modal cell.  From the code below we will displaying null to the user if the api returns null.

```
            unlinkedContracts !== '0' ? (<CellWithModal
                data={unlinkedContracts}
                openModal={openModal}
                modalType="unlinkedData"
                agencyData={{
                    agencyName,
                    agencyCode: code,
                    fiscalYear: selectedFy,
                    fiscalPeriod: selectedPeriod?.id,
                    type: 'Contract'
                }} />) : (<div className="generic-cell-content">{unlinkedContracts}</div>),
            unlinkedAssistance !== '0' ? (<CellWithModal
                data={unlinkedAssistance}
                openModal={openModal}
                modalType="unlinkedData"
                agencyData={{
                    agencyName,
                    agencyCode: code,
                    fiscalYear: selectedFy,
                    fiscalPeriod: selectedPeriod?.id,
                    type: 'Assistance'
                }} />) : (<div className="generic-cell-content">{unlinkedAssistance}</div>)
```

Reply:
Yes, they are handled in the helpers that are invoked in the models. I added tests for them. I can change those other properties, not actually dollar amount values, back to have the same treatment. Good catch... 

I have a thread pending though requesting confirmation if these should be `nullable` or not.

Reply:
I removed the changes to `unlinkedAssistance` and `unlinkedContracts` and confirmed `_gtasObligationTotal` handles null vaules [here](https://github.com/fedspendingtransparency/usaspending-website/blob/warmfix/dev-6952/show-dashdash-for-non-zero/src/js/helpers/aboutTheDataHelper.js#L155)

👌